contracts/DEXes/UniV3StablesJoint.sol

```diff
@@ -464,22 +454,25 @@ contract UniV3StablesJoint is NoHedgeJoint {
             int24 _minTick,
             int24 _maxTick
             ) external onlyVaultManagers {
-        IUniswapV3Pool(pool).burn(_minTick, _maxTick, uint128(_amount));
```

Comment:
both work but I think it is better to keep this down to the very minimum logic (so just leave it like it was before) in case collect fails. 

this is extra precaution but I think worth it.

Reply:
Burning position does not retrieve any tokens for the strategy but rather updates the position internally in Uni state and updates the `tokensOwed` part of the position info - only burning the position does not return any funds back to the strategy so if `collect` is bricked we will not be able to collect them